multimedia/v4l2camera/files/v4l2camera.init

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2017 OpenWrt.org
```

Comment:
please asisgn the copyright to yourself or just remove it

Reply:
ok just removed